src/lib.rs

```diff
@@ -2033,4 +2161,96 @@ mod test {
         assert_eq!(products[0], products[1]);
         assert_eq!(products[0], products[2]);
     }
+
+    mod formatter {
+        #[cfg(feature = "std")]
+        #[test]
+        fn format_as_decimal() {
+            use super::Ratio;
+
+            let data: [(u8, u8, &'static str); 60] = [
+                (1u8, 2u8, "0.5"),
```

Comment:
An example of what I've written in a comment - with 64 digits of precision maybe we should show up the following `0`'s.

Reply:
I believe this bit may be better to implement as `Fill + Align` functionality in the Display trait. We could also add leading zeroes, sign and other stuff later on top of current functionality.